### ISSUE TITLE
Rust | Add an option for snake_case to camelCase conversion on a per-struct basis instead of on a per-field basis

### ISSUE THREAD

**Issue by ritz078** [evestera/json_typegen#13]
The exact issue can be seen on https://github.com/ritz078/transform/issues/93

Add an option to do this
```Rust
#[derive(Serialize, Deserialize)]
#[serde(rename_all = "camelCase")]
struct RootInterface {
    some_object_with: String,
    a_lot_of_fields: String,
    which_quickly_becomes: String,
    hard_to_read: String,
}
```
instead of this
```Rust
#[derive(Serialize, Deserialize)]
struct RootInterface {
    #[serde(rename = "someObjectWith")]
    some_object_with: String,
    #[serde(rename = "aLotOfFields")]
    a_lot_of_fields: String,
    #[serde(rename = "whichQuicklyBecomes")]
    which_quickly_becomes: String,
    #[serde(rename = "hardToRead")]
    hard_to_read: String,
}
```

**Comment by evestera** [comment]
Implemented now in [release 0.4](https://github.com/evestera/json_typegen/releases/tag/v0.4.0). The option is also supported in the Kotlin codegen.

**Comment by ritz078** [comment]
Thank you ❤️
